### PR TITLE
MNT: Disable of_address for LTS

### LINKED ISSUE #7960: Add a helpful error message or way to pass in a key for from_address
As discussed in #7867, Google closed down non-API keyed access to the google maps address service.  This was fixed in master in #7918, but in a way that's not easily backportable. For the LTS branch we should do one of two things:

1. (Easy) Add a helpful error message that basically just says "sorry, google shut this down. You'll just have to manually get a lat/lon, or upgrade to astropy 3.1"
2. (Harder) Adapt the code in #7918 for passing a google API key, allowing users to get their own API keys and get back the functionality.

I have no strong opinion which one gets done other than that at least one of them gets done ASAP so that users don't get stuck with an obscure non-actionable error in the LTS branch any longer than necessary.

This should be implemented as a PR direct to the 2.x branch, *not* master, since #7918 fixed master in a not-easily-backportable way.

### PR DESCRIPTION
**Pull request by pllim**
Disable `of_address` for LTS. This is a backport of #7918. Fix #7960 .

p.s. I took the easy way out. 😬 🏃‍♀️ 🔥

### CODE REVIEW

File: astropy/coordinates/earth.py [hunk 1]
```diff
@@ -387,6 +387,11 @@ def of_address(cls, address, get_height=False):
         .. [2] https://developers.google.com/maps/documentation/elevation/intro
 
         """
+        # https://github.com/astropy/astropy/issues/7960
+        raise ValueError(
+            "Google has disabled usage of this API without API key. "
```
Comment (eteq) [inline astropy/coordinates/earth.py]:
```suggestion
            "Google has disabled usage of the interface necessary for of_address to work without an API key. "
```

**Comment by pllim** [comment]
Oh, the "apply suggestion" button is so convenient, thanks! p.s. I don't think 32-bit CircleCI failure is related.

**Comment by bsipocz** [comment]
The 32bit failures are OK, one is the usual one, the other is due to numpy 1.15.3. In either way I'm merging this, numpy will have a new release by the time we have a new LTS...

**Comment by bsipocz** [comment]
Thanks @pllim!

**Pull request merged**